pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -137,6 +150,14 @@ func (n *NodeLabeller) run() error {
 		n.logger.Infof("node-labeller cannot get new labels %s\n", err.Error())
 		return err
 	}
+	n.kvmInfoLabels.Lock()
+	defer n.kvmInfoLabels.Unlock()
+
+	if len(n.kvmInfoLabels.items) == 0 {
+		capScanner := NewCapScanner()
+		labels := capScanner.getLabels(true)
```

Comment:
do we need to pass in `true` here. i don't see any other instances of getLabels. why not make `true` the default behavior then?